### DISCUSSION TITLE
Починил YouTube на МГТС

### DISCUSSION THREAD

**Discussion by denius-dev** [dovecoteescapee/ByeDPIAndroid#175] [General]
Вариант 1
<details><summary>Click to expand</summary><p>

![image](https://github.com/user-attachments/assets/fa6522ef-9325-4ae2-8cd3-2b34fcb66c79)
![image](https://github.com/user-attachments/assets/6ce74676-56d0-4239-b727-7548607e060f)
![image](https://github.com/user-attachments/assets/46d31a8b-429c-4be8-8143-e93152c3104c)

</p></details>

Вариант 2 (актуальный у меня)
<details><summary>Click to expand</summary><p>

![image](https://github.com/user-attachments/assets/08cb8068-4733-407d-b3c5-337b5aa63b92)
![image](https://github.com/user-attachments/assets/0eba0776-8ace-42d3-9410-e2bba3638991)
![image](https://github.com/user-attachments/assets/ad1a178d-20f0-4c15-a4b3-89803bedc99f)
![image](https://github.com/user-attachments/assets/ad0a8481-7f65-4038-ad31-ee3bbe7bb048)

</p></details>

**Comment by denius-dev** [comment]
P.S. При открытии видео надо подождать секунд 10-15, потом без зависаний всё работает 👍

**Reply by hulio333** [reply]
не работает, меня только спас этот [пост](https://github.com/dovecoteescapee/ByeDPIAndroid/discussions/64#discussioncomment-10904410)

**Reply by denius-dev** [reply]
> не работает, меня только спас этот [пост](https://github.com/dovecoteescapee/ByeDPIAndroid/discussions/64#discussioncomment-10904410)

Возможно от провайдера к провайдеру по разному работает, у меня максимум пол минуты при открытии видео висит сначала, потом всё просто замечательно

**Reply by ReekySpring4** [reply]
Настройки помогли вернуть все в рабочее состояние, но desync metod я другой поставил, этот для меня не подошёл, спасибо за статью

**Comment by Pareq** [comment]
Не работает

**Comment by denius-dev** [comment]
Добавлю инфы, увеличивайте последний пункт, я уже 5000 выставил, помогает... я про фейковые пакеты UDP.

**Comment by Serega7774** [comment]
Не работает, ничего..

**Comment by smv2012** [comment]
юзайте тот, который 2раза-Bye. с простейшей строкой запустила на ревансед по вай-фай

**Comment by qiw1x** [comment]
на мгтс в мск все работает от души
